DGX_fair_use_guidelines.md

```diff
@@ -60,3 +60,70 @@ right:
 
 ---
 
+## Fair use of CPU resources
+
+Run:ai is designed to allocated GPU resources fairly, but CPU resources are not managed in the same way. It is up to 
+users to submit jobs with appropriate limits on CPU and CPU memory resources. A single job can use up the available 
+resources on a node and prevent all the other jobs from running, even if the GPU resources are available.
+
+Use the [--cpu-limit](https://docs.run.ai/v2.16/Researcher/cli-reference/runai-submit/#-cpu-limit-double) 
+and [--memory-limit](https://docs.run.ai/v2.16/Researcher/cli-reference/runai-submit/#-memory-limit-string) flags in 
+your `runai submit` command to set these limits. As a rule of thumb, divide the CPU and memory resources of the node by 
+the number of GPUs and set your limits according to how many GPUs you requested.
+
+Occasionally, you may need to run a CPU-only job on the server. Keep an eye on the general usage on the cluster and on 
```

Comment:
might be helpful to explain how target nodes can be specified here, which isn't documented somewhere

Reply:
Added tip